crates/bevy_core/src/time/time.rs

```diff
@@ -34,58 +49,171 @@ impl Time {
-            self.delta_seconds_f64 = self.delta.as_secs_f64();
             self.delta_seconds = self.delta.as_secs_f32();
+            self.delta_seconds_f64 = self.delta.as_secs_f64();
+            self.time_since_startup += self.delta.mul_f32(self.relative_speed);
+        } else {
+            self.first_update = Some(instant);
+            self.time_since_startup += (instant - self.startup).mul_f32(self.relative_speed);
         }
 
-        let duration_since_startup = instant - self.startup;
-        self.seconds_since_startup = duration_since_startup.as_secs_f64();
         self.last_update = Some(instant);
     }
 
-    /// The delta between the current tick and last tick as a [`Duration`]
+    /// The delta between the current and last update, as a [`Duration`].
     #[inline]
     pub fn delta(&self) -> Duration {
         self.delta
     }
 
-    /// The delta between the current and last tick as [`f32`] seconds
+    /// The delta between the current and last update, as [`f32`] seconds
     #[inline]
     pub fn delta_seconds(&self) -> f32 {
         self.delta_seconds
     }
 
-    /// The delta between the current and last tick as [`f64`] seconds
+    /// The delta between the current and last update, as [`f64`] seconds.
     #[inline]
     pub fn delta_seconds_f64(&self) -> f64 {
         self.delta_seconds_f64
     }
 
-    /// The time since startup in seconds
+    /// The *virtual* delta between each fixed update step, as a [`Duration`].
+    /// Note that the true delta between fixed update steps varies.
     #[inline]
-    pub fn seconds_since_startup(&self) -> f64 {
-        self.seconds_since_startup
+    pub fn fixed_delta(&self) -> Duration {
+        self.fixed_delta
     }
 
-    /// The [`Instant`] the app was started
+    /// The *virtual* delta between each fixed update step, as [`f32`] seconds.
+    /// Note that the true delta between fixed update steps varies.
     #[inline]
-    pub fn startup(&self) -> Instant {
-        self.startup
+    pub fn fixed_delta_seconds(&self) -> f32 {
+        self.fixed_delta_seconds
+    }
+
+    /// The *virtual* delta between each fixed update step, as [`f64`] seconds.
+    /// Note that the true delta between fixed update steps varies.
+    #[inline]
+    pub fn fixed_delta_seconds_f64(&self) -> f64 {
+        self.fixed_delta_seconds_f64
+    }
+
+    /// Changes [`Time::fixed_delta`]. Users are strongly cautioned against modifying
+    /// the step rate this way, except during startup.
+    /// Prefer [`Time::set_relative_speed`] to ensure that consumers of
+    /// [`Time::fixed_delta`] produce consistent behavior.
+    pub fn set_steps_per_second(&mut self, steps: f32) {
+        self.fixed_delta = Duration::from_secs_f32(1.0 / steps);
+        self.fixed_delta_seconds = self.fixed_delta.as_secs_f32();
+        self.fixed_delta_seconds_f64 = self.fixed_delta.as_secs_f64();
+    }
+
+    /// The rate that time advances relative to the real world, as [`f32`].
```

Comment:
```suggestion
    /// The rate at which (variable) time advances relative to the wall clock time, as [`f32`].
    /// This does not affect fixed update steps.
```

Reply:
Note while `time::delta()` varies, it's always 1:1 with the elapsed wall clock time. No seconds get skipped. If `relative_speed` is `1.0` the whole time, `time.time_since_startup()` and `time.real_time_since_startup()` will be equal.

Reply:
See my other comment, "wall clock" is wrong in retrospect. But so is "real world" in my opinion. I'd use "CPU time" instead:
```suggestion
    /// The rate at which the main simulation time advances relative to the CPU time, as [`f32`].
```